web/css/dashboard.css

```
@@ -0,0 +1,913 @@
+  * {
+    animation-duration: 0.01ms !important;
+    transition-duration: 0.01ms !important;
+  }
+}
+
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
```

Stylelint (17.14.1)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)